java/arcs/core/storage/DirectStore.kt

```diff
@@ -97,35 +100,34 @@ class DirectStore<Data : CrdtData, Op : CrdtOperation, T> /* internal */ constru
     fun getLocalData(): Data = synchronized(this) { localModel.data }
 
     override fun on(callback: ProxyCallback<Data, Op, T>): Int {
-        synchronized(proxyManager) {
+        synchronized(closed) {
```

Comment:
Is this legit? I assume it's autoboxing?

Reply:
It seems a little fishy..